concourse/pipelines/autodelete-cloudfoundry.yml

```diff
@@ -65,13 +60,11 @@ resources:
       versioned_file: bosh-CA.tar.gz
 
   - name: healthcheck-deployed
```

Comment:
I'm seeing some odd behaviour when concourse has some existing versions for this resource from the enum version. I'm wondering whether we should rename the resource (but not the versioned_file) to force concourse to start from a clean state.